mojaloop/values.yaml

```diff
@@ -8534,7 +8146,9 @@ ml-ttk-test-val-tp:
     testCasesZipUrl: *ttkGitUrl
     # testCasesPathInZip: *ttkGitPathGP
     # testCasesZipUrl: &ttkGitUrl https://github.com/mojaloop/testing-toolkit-test-cases/archive/refs/heads/release/v14.0.0.zip
-    testCasesPathInZip: testing-toolkit-test-cases-14.0.0/collections/hub/thirdparty
+    # testCasesPathInZip: testing-toolkit-test-cases-14.0.0/collections/hub/thirdparty
+    ## DELETE ME
```

Comment:
There are a few #delete me comments, should we delete them? @vijayg10 , @kleyow ..

Reply:
Happy to remove those if we merge this PR first: https://github.com/mojaloop/testing-toolkit-test-cases/pull/76

Reply:
We will be keeping using the [Testing-toolkit-test-cases v14.0.0-rc](https://github.com/mojaloop/testing-toolkit-test-cases/tree/release/v14.0.0-rc) release branch until we make an official release.